src/lpython/parser/parser.output

```diff
@@ -0,0 +1,525 @@
+Terminals unused in grammar
```

Comment:
This file should not be checked into git.

Reply:
@Abdelrahman-Kh-Fouad  Delete these files and add:
```
src/lpython/parser/parser.output
src/lpython/parser/parser.tab.cc
src/lpython/parser/parser.tab.hh
```
to .gitignore